### PR TITLE
done medical history checkbox

### PR DESCRIPTION
**Pull request by tracytrle**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,13 +3,14 @@
...
   "private": true,
   "dependencies": {
-    "@emotion/react": "^11.11.4",
-    "@emotion/styled": "^11.11.5",
+    "@emotion/react": "^11.12.0",
+    "@emotion/styled": "^11.12.0",
     "@gatsbyjs/reach-router": "^2.0.1",
     "@mui/icons-material": "^5.15.19",
-    "@mui/material": "^5.15.19",
+    "@mui/material": "^5.16.4",
```
Comment (travistrle) [inline package.json:10]: be careful upgrade library material. because it can break existing design

File: src/components/Forms/ClientInfo.js [hunk 2]
```diff
@@ -5,21 +5,23 @@ import DatePickerValue from "./DatePicker.Value";
 import DropoutMenu from "./DropoutMenu";
 import Typography from "@mui/material/Typography";
 import CurrentHealth from "./CurrentHealth";
-// import CustomizedCheckbox from "./CustomizedCheckbox";
+
```
Comment (travistrle) [inline src/components/Forms/ClientInfo.js]: Can we remove this empty line?

File: src/components/Forms/ClientInfo.js [hunk 3]
```diff
@@ -44,18 +46,33 @@ export default function FormPropsTextFields() {
           flexDirection: "row",
           justifyContent: "flex-start",
           alignItems: "center",
-          gap: 2, // Add spacing between the elements
+          // gap: 2, // Add spacing between the elements
```
Comment (travistrle) [inline src/components/Forms/ClientInfo.js]: If we don't need this, can we remove this comment?

File: src/components/Forms/CustomizedCheckbox.js [hunk 4]
```diff
@@ -1,24 +1,54 @@
...
+      <FormGroup row>
+        <FormControlLabel
+          control={
+            <Checkbox
+              sx={{
+                "&.Mui-checked": {
+                  color: "blue",
+                },
+              }}
+              checked={value === "yes"}
```
Comment (travistrle) [inline src/components/Forms/CustomizedCheckbox.js:25]: usually we prefer to define a constant string

**Review by travistrle — CHANGES_REQUESTED** [review]
can we rename `Forms` to `forms` ?

File: src/components/Forms/ClientInfo.js (hunk 2 above)
Comment (tracytrle) [reply src/components/Forms/ClientInfo.js]: OK I will do that later

File: src/components/Forms/ClientInfo.js (hunk 3 above)
Comment (tracytrle) [reply src/components/Forms/ClientInfo.js]: Ok

**Pull request merged**